### DISCUSSION TITLE
The test coverage is pretty low.  Is it in separate repository?

### DISCUSSION THREAD

**Discussion by siddharktw** [Stirling-Tools/Stirling-PDF#1156] [Q&A]
Unit and integration tests are very low.  
<img width="1780" alt="Screenshot 2024-05-03 at 12 30 35 PM" src="https://github.com/Stirling-Tools/Stirling-PDF/assets/166848282/1d6b28f0-3f23-4f06-983b-4301dc516afa">

**Comment by Frooodle** [comment]
There is a single unit test and it's in this repo 😂

**Reply by siddharktw** [reply]
I hope you don't mind if I raise a PR for improving the coverage.

**Reply by Frooodle** [reply]
Please do!

**Comment by jimdouk** [comment]
Hello! I was wondering if can I make some tasting to improve the test covering. 
*It would be very usefull for me to tell me which classes are most important to make test and which maybe is more easily to make.

**Reply by Frooodle** [reply]
So i recently added some post deployment tests in https://github.com/Stirling-Tools/Stirling-PDF/tree/main/cucumber to test the external dependency things like word to pdf which uses libreoffice.

As for JUnits, i would say the most used things are all the utils classes which should certainly be tested and ensured functionality remains good across updates
Outside of that it would just be the most used java PDF features, merge. split etc

But honestly we have no JUnits so any you can add anywhere would be good and appreciated
